src/XML/src/FMI3/fmi3_xml_parser.c

```diff
@@ -640,6 +640,86 @@ gen_fmi3_xml_set_attr_TYPEXX(uint32,  intXX)
         jm_log_error(context->callbacks, module, "Unable to allocate memory. File: %s, Line: %d", __FILE__, __LINE__);      \
     } while (0);
 
+/**
+ * Parses a string to an array of int values
+*/
+static int fmi3_xml_str_to_array_intXX(
```

Comment:
I will refactor this method next to remove code duplication between all the types.

Reply:
This was done in commit #03e4e6a